news/11545-move-content-trust-into-submodule

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Rework signature verification from pre-solve to post-solve process. (#11545)
```

Comment:
This is worth mentioning since it's not just an implementation detail.

```suggestion
* Rework signature verification from pre-solve to post-solve process. (#11545)

* Add a new `post-solves` plugin hook. (#11545)
```

Reply:
Verify signatures on to-be-installed packages instead of on all packages.

Reply:
Updated my suggestion